packages/smooth_app/lib/cards/data_cards/image_upload_card.dart

```diff
@@ -46,35 +46,25 @@ class _ImageUploadCardState extends State<ImageUploadCard> {
       if (!mounted) {
         return;
       }
-      final bool isUploaded = await uploadCapturedPicture(
+      await uploadCapturedPicture(
```

Comment:
@AshAman999 I don't really understand why you "await" here.
You only have to show the Snackbar, isn't-it ?

Reply:
workmanager does uses await before initialization and scheduling task so

Reply:
I mean, why do you have to wait here?
Since this is async, we don't have to wait for a response we never read